LPS-141976 | master

liuxingye opened the pull request:
https://issues.liferay.com/browse/LPS-141976

chaowang-123 committed:
LRQA-72210 Add a new test for localized web content

Commit:
COMMERCE-7491 BC Account widget

WYZ-Issac committed:
LRQA-72133 Add a locator for default language in Site Settings

WYZ-Issac committed:
LRQA-72133 Add macro for changing site default language

WYZ-Issac committed:
LRQA-72133 Add a test to verify that the Single Selection Form field added before default site language toggle can be loaded

WYZ-Issac committed:
LRQA-72133 Add a test to verify that the Single Selection Form field name and options can be added to non-English default site language site

Commit:
COMMERCE-8063 talend - onboarding documentation improvement

Commit:
COMMERCE-8063 talend - source formatted

victorg1991 committed:
LPS-144301 layout-seo-web Upgrade to testing library 12

victorg1991 committed:
LPS-144301 depot-web Upgrade to testing library 12 

- cleanup is done automatically after each test
- waitForElement is deprecated and removed
    - find* querys return a promise that resolve when the element is found (https://testing-library.com/docs/dom-testing-library/api-async#findby-queries)

pei-jung committed:
LPS-143952 This is not needed. AccountRoles will be removed when its corresponding Role is removed during company deletion. This logic is already handled in com.liferay.account.internal.model.listener.RoleModelListener

victorg1991 committed:
LPS-144301 document-library-video Upgrade to testing library 12 

- cleanup is done automatically after each test
- only forms with an accessible name have the role "form"
   Check: https://www.w3.org/TR/html-aria/#docconformance and https://github.com/testing-library/dom-testing-library/issues/937

victorg1991 committed:
LPS-144301 document-library-web Upgrade to testing library 12 

- cleanup is done automatically after each test
- only forms with an accessible name have the role "form"
   Check: https://www.w3.org/TR/html-aria/#docconformance and https://github.com/testing-library/dom-testing-library/issues/937
- waitForElement has been removed, now we can use find* variants, which return a promise, or waitFor

victorg1991 committed:
LPS-144301 flags-taglib Upgrade to testing library 12 

- cleanup is done automatically after each test
- only forms with an accessible name have the role "form"
   Check: https://www.w3.org/TR/html-aria/#docconformance and https://github.com/testing-library/dom-testing-library/issues/937
- waitForElement has been removed, now we can use find* variants, which return a promise, or waitFor

victorg1991 committed:
LPS-144301 friendly-url-taglib Upgrade to testing library 12 

- cleanup is done automatically after each test
- waitForElement has been removed, now we can use find* variants, which return a promise, or waitFor

victorg1991 committed:
LPS-144301 layout-content-page-editor-web Upgrade to testing library 12

- cleanup is done automatically after each test
- waitForElement has been removed, now we can use find* variants, which return a promise, or waitFor

victorg1991 committed:
LPS-144301 portal-workflow-instance-tracker-web Upgrade to testing library 12 

- cleanup is done automatically so render component beforeEach test instead of render once

victorg1991 committed:
LPS-144301 segments-web Upgrade to testing library 12 

- cleanup is done automatically so render component beforeEach test instead of render once
- wait function is removed, replaced by waitFor

victorg1991 committed:
LPS-144301 analytics-reports-web Upgrade to testing library 12 

- cleanup is done automatically so render component beforeEach test instead of render once
- wait function is removed, replaced by waitFor

victorg1991 committed:
LPS-144301 portal-search-tuning-rankings-web Upgrade to testing library 12 

- cleanup is done automatically so render component beforeEach test instead of render once
- waitForElement has been removed now we can use find* variants, which return a promise, or waitFor

victorg1991 committed:
LPS-144301 portal-search-tuning-synonyms-web Upgrade to testing library 12 

- cleanup is done automatically

victorg1991 committed:
LPS-144301 segments-experiment-web Upgrade to testing library 12 

- cleanup is done automatically
- wait* function are removed and replaced by waitFor

victorg1991 committed:
LPS-144301 content-dashboard-web Upgrade to testing library 12 

- cleanup is done automatically
- wait* function are removed and replaced by waitFor or find* query variant
- label for checkbox was not correctly set, aria-labelledby should receive an id, not the label itself

victorg1991 committed:
LPS-144301 commerce-shop-by-diagram-web Upgrade to testing library 12 

- cleanup is done automatically
- wait* function are removed and replaced by waitFor or find* query variant

victorg1991 committed:
LPS-144301 data-engine-taglib Upgrade to testing library 12 

- cleanup is done automatically
- wait* function are removed and replaced by waitFor

victorg1991 committed:
LPS-144301 dynamic-data-mapping-form-field-type Upgrade to testing library 12

- cleanup is done automatically
- wait* function are removed and replaced by waitFor and variants of find* queries

victorg1991 committed:
LPS-144301 data-engine-js-components-web Upgrade to testing library 12 

- cleanup is done automatically
- wait* function are removed and replaced by waitFor and variants of find* queries
- Changed button name since it required "-"

victorg1991 committed:
LPS-144301 portal-workflow-metrics-web Upgrade to testing library 12

- cleanup is done automatically. Set RTL_SKIP_AUTO_CLEANUP=true since this all the test rely on this
- find* variants return a promise

victorg1991 committed:
LPS-144301 Skip these failing tests for now

victorg1991 committed:
LPS-144301 commerce-frontend-js Upgrade to testing library 12

 - cleanup is done automatically
 - wait* function are removed and replaced by waitFor and variants of find* queries

victorg1991 committed:
LPS-144301 batch-planner-web Upgrade to testing library 12

 - cleanup is done automatically
 - wait* function are removed and replaced by waitFor and variants of find* queries
 - getByRole('textbox') doesn't return inputs of type file
 - fakeTimers doesn't play very well with a mix of setInterval and promises so I've simplified it a bit to not rely on them

thektan committed:
LPS-143640 search-experiences-web: Remove unused components

thektan committed:
LPS-143640 search-experiences-web: Remove framework selector in Add Blueprint Modal

Blueprint is only created with "All Clauses" framework

thektan committed:
LPS-143640 search-experiences-web: Update layout styling

- Add vertical navigation to query builder tab
- Increase width of vertical nav content
- Stack columns on smaller screens

BryanEngler committed:
LPS-144380 search-experiences-rest-impl: Add schemaVersion

https://issues.liferay.com/browse/LPS-144380

Co-authored-by: Bryan Engler <bryan.engler@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

BryanEngler committed:
LPS-144380 buildREST

https://issues.liferay.com/browse/LPS-144380

Co-authored-by: Bryan Engler <bryan.engler@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

BryanEngler committed:
LPS-144380 search-experiences-service: Add schemaVersion

https://issues.liferay.com/browse/LPS-144380

Co-authored-by: Bryan Engler <bryan.engler@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

BryanEngler committed:
LPS-144380 buildService

https://issues.liferay.com/browse/LPS-144380

Co-authored-by: Bryan Engler <bryan.engler@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

BryanEngler committed:
LPS-144380 buildService

https://issues.liferay.com/browse/LPS-144380

Co-authored-by: Bryan Engler <bryan.engler@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

BryanEngler committed:
LPS-144380 search-experiences-rest-impl: Get schema version and pass it to service call

https://issues.liferay.com/browse/LPS-144380

Co-authored-by: Bryan Engler <bryan.engler@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

BryanEngler committed:
LPS-144380 search-experiences-service: Add schema version arguments to service methods

https://issues.liferay.com/browse/LPS-144380

Co-authored-by: Bryan Engler <bryan.engler@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

oliv-yu committed:
LPS-143640 search-experiences-web: Implement Query Settings tab

- Create QueryElements and QuerySettings + add radio to QuerySettings
- Add Search Framework Indexer Clauses toggle to QuerySettings
- Set up Clause Contributors sidebar
- Rename showSidebar to showAddSXPElement for clarity
- Rename QueryElements to QuerySXPElements
- Clean up props in QueryBuilderTab
- Add management toolbar + styling to ClauseContributorsSidebar
- Close sidebars when navigating to other tab or opening another sidebar
- Remove Clause Contributors Tab
- Adjust panel header height

BryanEngler committed:
LPS-144380 search-experiences-service: Set schema version in local service

https://issues.liferay.com/browse/LPS-144380

Co-authored-by: Bryan Engler <bryan.engler@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

BryanEngler committed:
LPS-144380 search-experiences-rest-impl: Set in DTO

https://issues.liferay.com/browse/LPS-144380

Co-authored-by: Bryan Engler <bryan.engler@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

BryanEngler committed:
LPS-144380 search-experiences-service: Set schema version in CompanyModelListener

https://issues.liferay.com/browse/LPS-144380

Co-authored-by: Bryan Engler <bryan.engler@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

BryanEngler committed:
LPS-144380 search-experiences-test: Update tests

https://issues.liferay.com/browse/LPS-144380

Co-authored-by: Bryan Engler <bryan.engler@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

weller0246 committed:
LPS-144427 Extend existing macros to be compatible with 7.0 upgrade tests

weller0246 committed:
LPS-144427 Fix upgrade test for 7.0

Commit:
LPS-144586 Include Team Extranet Test Properties

Commit:
LPS-144586 Create Team Extranet macro, path and testcase

Commit:
LPS-144586 SF

oliv-yu committed:
LPS-143640 search-experiences-web: Update frontend tests for query builder and new clause contributors sidebar

oliv-yu committed:
LPS-143640 search-experiences-web: Rename settings tab to configuration

oliv-yu committed:
LPS-143640 search-experiences-web: Put helptext inside QuerySettings

oliv-yu committed:
LPS-143640 search-experiences-web: Remove unused applyIndexerClause function

oliv-yu committed:
LPS-143640 search-experiences-web: Use frameworkConfig's clauseContributorsIncludes/Excludes instead of 'enabled' state

oliv-yu committed:
LPS-143640 search-experiences-web: Clarify names and alphabetize functions

oliv-yu committed:
LPS-143640 search-experiences-web: Keep track of sidebar visibility by using just one state

oliv-yu committed:
LPS-143640 portal-language-lang: Language keys for SXP redesign

victorg1991 committed:
LPS-144301 Use latest npm-scripts version

victorg1991 committed:
LPS-144301 search-experiences-web Upgrade to testing library 12

    - cleanup is done automatically

chaowang-123 committed:
LRQA-41068 Add the test for web content can be rendered with the default template

oliv-yu committed:
LPS-143640 search-experiences-web: Prevent excessive API calls by fetching in EditSXPBlueprintForm

oliv-yu committed:
LPS-143640 portal-language-lang: gradlew buildLang

Commit:
LPS-144557 Fix Okta session approach

Commit:
LPS-144557 Fix status none for accountSubscriptions cards

Commit:
LPS-144586 add Team Extranet property on testcases

sez11a committed:
COMMERCE-8063 Wordsmithing.

epark18 committed:
COMMERCE-7083 Create test to verify Mapped Products table saves pin info

epark18 committed:
COMMERCE-7083 Add paths for Mapped Products table

epark18 committed:
COMMERCE-7083 Update addPin macro in appropriate file to move pins conditionally

epark18 committed:
COMMERCE-7027 Add skip portal instance for RemovePin

ling-alan-huang committed:
LPS-144888 Use isSet() to check whether a variable was set, instead of an old pattern by using contains()

ling-alan-huang committed:
LPS-144888 Remove redundant parentheses

ling-alan-huang committed:
LPS-144888 Auto SF

ling-alan-huang committed:
LPS-144888 Remove redundant code

Commit:
LPS-144529 add Team Extranet testcases

Commit:
LPS-144529 SF

Commit:
LPS-144197 Update Colorful Head and Insurance FAQ fragments to Mobile and Tablet View

Commit:
LPS-144800 Update header to Mobile and Tablet view

hhuijser committed:
LPS-105380 Generated

wcao20170619 committed:
LPS-143497 search-experiences-test:fix integration test failures

https://issues.liferay.com/browse/LPS-143497

michaelhashimoto committed:
LRCI-2605 Use the JSONObject to determine the hash code since it has more factors

victorg1991 committed:
LPS-144301 journal-web Upgrade to testing library 12

    - cleanup is done automatically after each test
    - waitForElement has been removed, now we can use find* variants, which return a promise, or waitFor
    - The case of the inputFile is a bit special, since it is an input of type file and hidden, hence, it won't be returned when looking for textbox role

kevenleone committed:
LPS-144197 Add responsiveness to Raylife Public Footer

Co-authored-by: Rosinaldo Guedes <rosinaldo.guedes@liferay.com>
Co-authored-by: Wellington Barbosa <wellington.barbosa@liferay.com>"

kevenleone committed:
LPS-144197 Add Package.json

Commit:
LPS-144865 Fix card grid padding

Commit:
LPS-144865 Fix font-weight-normal to 400

arboliveira committed:
LPS-144153 search-experiences-rest-test: Zero Results test requires response string to assert presence of "hits" section

https://issues.liferay.com/browse/LPS-144153

arboliveira committed:
LPS-144380 search-experiences-rest-test: Update tests

https://issues.liferay.com/browse/LPS-144380

Co-authored-by: Bryan Engler <bryan.engler@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

hhuijser committed:
LPS-105380 SF fails with latest portal-kernel

thektan committed:
LPS-143640 search-experiences-web: Query settings panels expanded by default

Also:
- Removes extra margin-bottom in panel body
- CSS class rename for more clarity

thektan committed:
LPS-143640 search-experiences-web: Selecting enable all was closing preview sidebar too

thektan committed:
LPS-143640 search-experiences-web: Default to all selected asset types

An empty array means all selected asset types are searched

thektan committed:
LPS-143640 search-experiences-web: Formatting

Sorting method, alphabetizing, and variable renames fo readability

Hanlf committed:
LPS-144945 Use macro to add site

michaelhashimoto committed:
LRCI-2605 Ensure that fix URL can be run multiple times against a URL or snippet

thektan committed:
LPS-143567 search-experiences-web: Fixes JS error when custom json includes uiConfiguration property

The error was occurring because upon save and revisiting, the custom json element was attempting to convert into an element with input fields.

lipusz committed:
LPS-144910 search-experiences-service: Fix invalid property name in "Boost Contents in a Category for New User Accounts"

Signed-off-by: Tibor Lipusz <tibor.lipusz@liferay.com>

lipusz committed:
LPS-144951 search-experiences-service: Fix invalid property name in "Scheduling Aware"

https://issues.liferay.com/browse/LPS-144951

Signed-off-by: Tibor Lipusz <tibor.lipusz@liferay.com>

hhuijser committed:
LPS-143349 Add logic to enforce running checks in specific order

michaelhashimoto committed:
LRCI-2605 Replace %2B for with a plus if it exists

kevenleone committed:
LPS-144994 Page Responsiviness to Small Devices

Co-authored-by: Silas Silva <silas.silva@liferay.com>

kevenleone committed:
LPS-144995 Get Window Dimensions and his device name

Co-authored-by: Silas Silva <silas.silva@liferay.com>

kevenleone committed:
LPS-144995 Use an existing MoreInfo Button

Co-authored-by: Silas Silva <silas.silva@liferay.com>

kevenleone committed:
LPS-144994 Modal adjust styles

Co-authored-by: Silas Silva <silas.silva@liferay.com>

kevenleone committed:
LPS-144993 Create WebContentProvider to handle Tip Container for mobile screen

Co-authored-by: Silas Silva <silas.silva@liferay.com>

kevenleone committed:
LPS-144993 Create WebContentProvider to handle Tip Container for mobile screen

Co-authored-by: Silas Silva <silas.silva@liferay.com>

kevenleone committed:
LPS-144994 Update Get a Quote page for responsiveness

Co-authored-by: Silas Silva <silas.silva@liferay.com>

wcao20170619 committed:
LPS-143527 search-experiences-service:add try-catch blocks

https://issues.liferay.com/browse/LPS-143527

kevenleone committed:
LPS-144994 Use a Liferay global variable as const to avoid try catch everytime

Co-authored-by: Silas Silva <silas.silva@liferay.com>

kevenleone committed:
LPS-144994 Adapt Get a Quote Page to Small Devices

Co-authored-by: Silas Silva <silas.silva@liferay.com>

kevenleone committed:
LPS-144994 SF

brittneyq committed:
LRCI-2608 Add clear method

thektan committed:
LPS-143640 search-experiences-web: Show preview management bar with 0 results

Refresh button and view raw response are still applicable when there are no results

Commit:
LPS-144927 Show the project name instead project code

Commit:
LPS-144927 SF

thektan committed:
LPS-143640 search-experiences-web: Fix configuration sheet width

It was spanning the whole width, this change decreases it back to how it originally was.

peerkar committed:
LPS-145015 search-experiences-service: Fix value extraction and parameter name-definition mappings

peerkar committed:
LPS-145015 portal-language-lang: New language key

peerkar committed:
LPS-145015 portal-language-lang: gw buildLang

peerkar committed:
LPS-145016 search-experiences-service: User age and email domain parameter name fix

peerkar committed:
LPS-145018 search-experiences-service: Rename Publication ID parameter definition

brianchandotcom committed:
LPS-145015 never used again

brianchandotcom committed:
LPS-145015 SF

brianchandotcom committed:
LPS-145018 sort

brianchandotcom committed:
LRCI-2605 prep next

brianchandotcom committed:
LPS-143527 consistency

brianchandotcom committed:
LPS-144994 upload

brianchandotcom committed:
LPS-143349 prep next

brianchandotcom committed:
LPS-143349 prep next

brianchandotcom committed:
LPS-144380 Sort

brianchandotcom committed:
LPS-144380 Sort

brianchandotcom committed:
LPS-144380 SF

brianchandotcom committed:
LPS-144380 regen

brianchandotcom committed:
Bump up because of 7.3.x 203c4325583de3236532b72596c5b55bad585b22

brianchandotcom committed:
LPS-144888 prep next

brianchandotcom committed:
LPS-144888 prep next

liuxingye committed:
LPS-141976 Assert history icon before typing custom URL

brianchandotcom committed:
artifact:ignore portal-store-azure 5.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-store-azure 5.0.0 artifact properties

brianchandotcom committed:
artifact:ignore sharepoint-soap-repository 5.0.0 prep next

brianchandotcom committed:
artifact:ignore sharepoint-soap-repository 5.0.0 artifact properties

liuxingye committed:
LPS-141976 Add macro for going to blogs friendlyURL history

liuxingye committed:
LPS-141976 Add paths for viewing blogs friendlyURLs in history

liuxingye committed:
LPS-141976 Add macros for viewing blogs friendlyURLs in history

liuxingye committed:
LPS-141976 Add macro for restoring blogs old friendly URLs

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

liuxingye commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 4 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/26445341f1a9d1d4fa9dd05ee88932b57f8844f6">26445341f1a9d1d4fa9dd05ee88932b57f8844f6</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/liuxingye/liferay-portal/tree/master-lps-141976">master-lps-141976</a><br/>Branch GIT ID: <a href="https://github.com/liuxingye/liferay-portal/commit/3851e89cf5d24b3dfb7ae698ec7baa7fb1e2aea6">3851e89cf5d24b3dfb7ae698ec7baa7fb1e2aea6</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-34.liferay.com/job/test-portal-source-format/567/"><strike><strong>test-portal-source-format</strong></strike></a></li></ul></div><h5>For more details click <a href="https://test-1-34.liferay.com/userContent/jobs/test-portal-source-format/builds/567/jenkins-report.html">here</a>.</h5><pre><code>     [java] java.lang.Exception: Found 1 formatting issues:
     [java] 1: ./portal-web/test/functional/com/liferay/portalweb/macros/BlogsEntry.macro expected:&lt;...odal#CLOSE_BUTTON");[]
     [java]   \}
     [java] 
     [java]   macro removeSm...&gt; but was:&lt;...odal#CLOSE_BUTTON");[    ]
     [java]   \}
     [java] 
     [java]   macro removeSm...&gt;
     [java] 
     [java]   at com.liferay.source.formatter.SourceFormatter.format(SourceFormatter.java:403)
     [java]   at com.liferay.source.formatter.SourceFormatter.main(SourceFormatter.java:249)
[stopwatch] [run.batch.test.action: 1:45.656 sec]
     [echo] The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/portal-impl/build.xml:555: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/portal-impl/build.xml:695: Java returned: 1
      [get] Getting: http://test-1-34/job/test-portal-source-format/567//consoleText
      [get] To: /opt/dev/projects/github/liferay-portal/20220109234329570.txt
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/20220109234329570.txt
  [typedef] Could not load definitions from resource org/apache/maven/artifact/ant/antlib.xml. It could not be found.
  [taskdef] Could not load definitions from resource org/jacoco/ant/antlib.xml. It could not be found.
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null1550575510.properties

merge-test-results:
    [mkdir] Created dir: /opt/dev/projects/github/liferay-portal/test-results
[beanshell] Truncating errors in /opt/dev/projects/github/liferay-portal/portal-impl/test-results/TEST-JenkinsLogAssertTest.xml
[junitreport] Processing /opt/dev/projects/github/liferay-portal/test-results/TESTS-TestSuites.xml to /tmp/null1489686658
[junitreport] Loading stylesheet jar:file:/opt/java/ant/lib/ant-junit.jar!/org/apache/tools/ant/taskdefs/optional/junit/xsl/junit-frames.xsl
[junitreport] Transform time: 437ms
[junitreport] Deleting: /tmp/null1489686658
     [echo] A report with all the test results can be found at test-results/html/index.html.
    [mkdir] Created dir: /opt/dev/projects/github/liferay-jenkins-ee/test-results
     [copy] Copying 1 file to /opt/dev/projects/github/liferay-jenkins-ee/test-results
     [echo] run.batch.test.tear.down.start.timestamp: 01-09-2022 23:43:32:157 PST
[stopwatch] [run.batch.test.tear.down: 0.000 sec]</code></pre></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-34.liferay.com/job/test-portal-source-format/567/">test-portal-source-format#567</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-34.liferay.com/userContent/jobs/test-portal-source-format/builds/567/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liuxingye/liferay-portal/pull/506">liuxingye#506</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1598032402">[master] ci:test:sf - liuxingye &gt; liuxingye - PR#506 - 2022-01-09[23:40:17]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-34.liferay.com/job/publish-testray-report/1972/">publish-testray-report#1972</a><br/></div></div>

liuxingye committed:
LPS-141976 Add macro for removing old friendly URL

liuxingye committed:
LPS-141976 Add tests for managing history of blogs friendly URL

liuxingye commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/26445341f1a9d1d4fa9dd05ee88932b57f8844f6">26445341f1a9d1d4fa9dd05ee88932b57f8844f6</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/liuxingye/liferay-portal/tree/master-lps-141976">master-lps-141976</a><br/>Branch GIT ID: <a href="https://github.com/liuxingye/liferay-portal/commit/492865a6f843556011e0f94f3b08944bc7b3e94e">492865a6f843556011e0f94f3b08944bc7b3e94e</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-31.liferay.com/job/test-portal-source-format/1480/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-31.liferay.com/userContent/jobs/test-portal-source-format/builds/1480/jenkins-report.html">here</a>.</h5></details></html>

Commit:
LPS-144676 Add macro to detach token

Commit:
LPS-144676 Add case to detach linked color token in page editor

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-31.liferay.com/job/test-portal-source-format/1480/">test-portal-source-format#1480</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-31.liferay.com/userContent/jobs/test-portal-source-format/builds/1480/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liuxingye/liferay-portal/pull/506">liuxingye#506</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1598032520">[master] ci:test:sf - liuxingye &gt; liuxingye - PR#506 - 2022-01-09[23:50:52]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-31.liferay.com/job/publish-testray-report/5131/">publish-testray-report#5131</a><br/></div></div>

Jeremy-Ch committed:
LPS-145024 Add property to skip running on release bundle

Commit:
LPS-144676 Modify locator to make it reusable

Commit:
LPS-144676 Add case to view color token changes applied to usages

Jeremy-Ch committed:
LPS-145026 Remove the unnecessary steps

Jeremy-Ch committed:
LPS-145026 Update the paramter value

liulukiwi committed:
LPS-144529 Use existing macros to add team extranet site and remove db property

joseabelenda committed:
LPS-145036 add fields to fix ObjectDefinition tests

rafaellajordao committed:
LRQA-72477 remove Form.modifyModuleDefinitionTimeout(); to setup

kevenleone committed:
LPS-145043 Improve Liferay API in Remote App

brianchandotcom committed:
LPS-145043 upload

kyle-miho committed:
LRQA-72519 Regen portal-osgi-configuration.properties

lipusz committed:
LPS-145058 search-experiences-service: Remove the Commerce element from the MVP

This is requiring more development, will be re-added

https://issues.liferay.com/browse/LPS-145058

Signed-off-by: Tibor Lipusz <tibor.lipusz@liferay.com>

kevenleone committed:
LPS-144888 Use Liferay as a local variable to avoid errors when use Remote App out of Liferay Portal

brianchandotcom committed:
LPS-143640 LPS-143567 wordsmith

brianchandotcom committed:
LPS-143640 LPS-143567 regen

Commit:
LPS-144684 search-experiences-rest-impl: Add jackson-core on build.gradle

brianchandotcom committed:
LPS-144888 SF
